packages/web/src/components/inbox-unavailable-modal/InboxUnavailableModal.module.css

```diff
@@ -0,0 +1,27 @@
+.modalBody {
+  max-width: 480px;
+}
+
+.icon path {
+  fill: var(--neutral);
+}
+
+.nameAndBadge {
+  display: inline-flex;
+  gap: var(--unit-1);
+  cursor: pointer;
+}
+
+.content {
+  line-height: 130%;
+  font-size: var(--font-l);
+  font-weight: var(--font-medium);
+}
+
+.footer {
+  padding: 0 var(--unit-6) var(--unit-6);
```

Comment:
FYI @dylanjeffers a good case study re: Harmony, I think this file should be made wholly unnecessary, but especially the `.content`, `.footer` and `.icon` classes

Reply:
Thanks for heads up! What would you prefer to see instead? Having a box component that handles this, or using flex gap or something?

Reply:
None of the above - all of these styles could easily be made as defaults to the relevant modal components

This is such a basic modal that it should just work out of the box

Reply:
Eg. Button should have a variant that's full width, ModalFooter should have no top padding but the same horizontal and bottom padding as ModalContent, and ModalContent should have the proper font style defaults. Icons in the ModalHeader should fill with the appropriate colors (whether that's at the icon level or the modal level idk) and perhaps the modal width should be a prop?

Reply:
You could also make the case (a la MUI) that `ModalHeader` and `ModalFooter` could be reusable components that render the common cases for us and just accept callbacks for things like clicking the primary action etc.

Reply:
Oh yeah 💯